### PR TITLE
<MarketingLayout/> - add badge and hiddenBadge props

### PR DESCRIPTION
**Pull request by yogevitz**
### 🔦 Summary

add `badge` and `hiddenBadge` props.

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- [x] 👨‍🎨 UX change is approved by the Design System UX @milkyfruit 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6273

File: src/MarketingLayout/MarketingLayout.st.css [hunk 1]
```diff
@@ -27,6 +27,16 @@
...
 
+.root:badge {
+  position: relative;
+  padding-top: 36px;
+}
+
+.root:hiddenBadge {
+  position: relative;
+  padding-top: 36px;
+}
```
Comment (EgoziE) [inline src/MarketingLayout/MarketingLayout.st.css:38]:
```suggestion
.root:badge,
.root:hiddenBadge {
  position: relative;
  padding-top: 36px;
}
```

File: src/MarketingLayout/test/MarketingLayout.tsx [hunk 2]
```diff
@@ -21,6 +22,7 @@ function marketingLayoutWithAllProps() {
       actions={<div />}
       title="title"
       description="description"
+      badge={<Badge size="small">badge</Badge>}
```
Comment (EgoziE) [inline src/MarketingLayout/test/MarketingLayout.tsx:25]: Missing hiddenBadge prop

```suggestion
      badge={<Badge size="small">badge</Badge>}
      hiddenBadge
```

**Review by EgoziE — COMMENTED** [review]
@milkyfruit Ready for UX review

**Review by milkyfruit — APPROVED** [review]
@EgoziE PR looks great!

**Pull request merged**